Branch-SDK-TestBed/src/io/branch/branchandroiddemo/MainActivity.java

```diff
@@ -266,10 +268,12 @@ public void onShareLinkDialogDismissed() {
 
                             @Override
                             public void onLinkShareResponse(String sharedLink, String sharedChannel, BranchError error) {
+                                branchUniversalObject.userCompletedAction(BranchUniversalObject.BUO_USER_ACTIONS.SHARE_COMPLETED);
                             }
 
                             @Override
                             public void onChannelSelected(String channelName) {
+                                branchUniversalObject.userCompletedAction(BranchUniversalObject.BUO_USER_ACTIONS.SHARE_STARTED);
```

Comment:
@ahmednawar This is called only from the testbed app not from SDK. So it wont affect other apps